src/DotNetWorker.Grpc/DotNetWorker.Grpc.csproj

```diff
@@ -40,7 +40,7 @@
     we use the legacy Grpc.Core package instead, due to limitations with the current 
     Grpc.Net.Client implementation-->
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="Grpc.Core" Version="2.45.0" />
+    <PackageReference Include="Grpc.Core" Version="2.46.6" />
```

Comment:
Even though a project reference to "**Worker.Extensions.Rpc.csproj**" was added to this project (line 58 below), we still need to keep these explicit package references for Grpc packages (Grpc.Core, Grpc.Net.Client, Grpc.Net.ClientFactory) because DotNetWorker.Grpc project still have net5 support where Worker.Extensions.Rpc does not.

Reply:
Closing this comment as thus change is not valid anymore.